cmsplugin_feed_ai/management/commands/test_social_feeds.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from cmsplugin_feed_ai.feed import _fetch_profile_feed
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Test feed fetching"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-p",
+            "--profile",
+            action="store",
+            dest="profile_name",
+            default="POTUS",
+            help="Profile name to be used. Defaults to POTUS.",
+        )
+
+        parser.add_argument(
+            "-l",
+            "--limit",
+            action="store",
+            dest="limit",
+            default=2,
+            help="Limit amount of items fetched from each service.",
+        )
+
+    def handle(self, *args, **options):
+        profile_name = options["profile_name"]
+        limit = int(options["limit"])
+        self.output_feeds(profile_name, limit)
+
+    def output_feeds(self, profile_name, limit):
+        twitter_feed = _fetch_profile_feed("twitter", profile_name, limit)
```

Comment:
Once you've refactored the fetchers into a dict, this can just use that -- and then we're practically out of places where `twitter` and `facebook` are hard-coded! :)